ibm_mq/datadog_checks/ibm_mq/data/conf.yaml.example

```diff
@@ -85,6 +85,11 @@ instances:
     #
     # auto_discover_queues: false
 
+    ## @param mqcd_version - integer - optional - default: 6
+    ## Which API version to use. Supported values are 1 to 9 including.
```

Comment:
Why don't we support 10 to 12?

Reply:
Because they're not defined in pymqi

Reply:
I've added that supported versions to the README